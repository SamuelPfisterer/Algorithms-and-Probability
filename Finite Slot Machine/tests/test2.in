30
77 393
1 3 5 3 3 10 7 8 5 6 9 7 3 2 10 0 1 2 5 7 3 0 0 1 9 1 7 0 9 6 10 4 1 2 2 1 1 6 4 6 4 5 10 6 4 3 6 9 10 8 7 8 9 4 10 3 10 0 1 7 10 7 5 10 2 1 6 4 4 5 5 10 8 7 0 8 7
54 528
2 2 2 2 1 2 0 1 0 1 2 1 0 0 0 1 1 2 0 2 2 1 2 1 0 1 1 2 484 0 0 0 0 0 0 0 1 0 2 0 0 1 0 0 0 2 0 0 2 2 1 0 0 1
66 316
7 8 9 10 8 6 0 0 8 10 3 9 3 10 2 4 4 5 3 4 10 7 2 10 1 4 8 0 3 4 0 2 3 7 6 2 3 3 1 1 9 10 9 7 5 7 1 2 2 9 0 8 3 6 1 7 4 5 4 1 4 4 3 2 8 5
72 471
2 0 1 2 1 1 2 0 2 2 0 2 1 2 1 2 0 0 1 2 1 0 2 1 0 0 0 2 1 0 2 1 2 0 0 2 1 1 2 1 392 2 0 2 1 1 1 2 2 0 1 2 0 0 1 0 1 2 2 0 1 2 2 2 1 2 0 0 2 0 2 2
10 17
2 1 0 3 1 0 3 3 2 2
93 379
2 0 1 2 1 1 2 1 2 2 0 1 1 2 1 1 0 278 1 1 1 1 0 0 2 0 2 0 2 1 1 2 2 1 2 2 0 1 2 1 2 1 1 2 1 0 0 2 1 2 0 0 0 0 1 2 2 2 0 0 0 2 1 1 0 2 0 1 1 1 0 0 2 0 1 1 2 2 2 0 2 2 1 0 2 1 1 1 1 2 2 2 1
81 401
8 6 0 4 1 8 0 2 3 4 6 1 6 8 3 7 7 2 3 5 6 8 7 7 4 1 6 5 8 6 9 9 2 7 6 7 7 8 6 5 10 10 5 6 3 0 3 4 3 8 2 7 1 1 7 10 0 8 2 4 6 5 9 8 1 5 6 0 3 9 1 10 1 9 2 1 9 6 4 3 1
10 12
1 10 0 0 0 0 0 0 0 1
10 13
1 2 1 1 3 0 1 2 1 1
56 382
1 2 2 1 0 2 1 2 0 1 2 0 1 2 0 1 2 0 2 1 1 0 0 0 320 1 1 1 1 2 1 0 1 2 2 2 1 1 1 1 0 1 2 2 2 2 1 2 0 0 2 2 1 0 2 1
59 292
10 8 7 5 8 5 2 10 8 2 4 3 0 0 6 6 6 9 2 4 3 2 2 6 0 7 4 5 1 9 3 8 3 7 6 1 2 8 6 8 4 6 7 9 6 10 4 7 10 1 2 7 0 5 9 0 3 3 3
10 18
2 1 1 2 1 2 1 0 0 8
10 15
3 3 0 2 0 1 0 3 0 3
63 316
0 1 2 1 0 2 2 1 0 0 2 2 2 0 0 0 0 1 1 0 2 2 1 2 0 2 1 1 1 0 1 2 1 1 250 0 1 1 0 1 2 0 0 2 2 1 2 1 1 0 2 1 2 0 2 0 2 2 2 1 2 1 1
10 14
2 1 2 3 3 0 0 1 1 1
75 281
1 0 1 1 2 0 1 1 1 0 1 1 1 1 0 0 2 1 1 2 2 2 2 2 1 1 1 1 2 0 0 0 0 0 2 1 2 2 1 0 0 0 2 0 2 218 1 0 1 1 1 1 1 0 1 0 0 1 0 1 1 1 0 0 0 0 2 2 0 0 2 0 1 0 1
54 283
5 9 2 9 6 1 3 2 5 1 3 3 10 7 10 10 9 3 9 5 6 6 0 4 0 10 0 9 2 6 10 0 3 6 4 2 10 5 10 6 2 5 0 3 5 10 7 6 8 6 8 2 10 0
50 530
1 1 2 1 1 0 2 2 1 0 0 1 2 0 2 0 0 1 0 0 1 2 1 2 482 1 0 0 2 0 2 2 2 0 1 0 2 0 0 2 2 1 0 0 1 1 2 2 1 1
59 249
8 9 6 9 0 3 1 0 7 3 2 1 4 4 4 2 7 6 4 1 6 5 10 3 3 8 7 0 9 3 0 5 10 2 1 0 9 2 2 5 3 5 4 1 10 6 0 0 0 7 6 1 10 7 4 9 2 0 3
10 21
2 1 2 2 0 1 10 1 0 2
50 190
0 5 4 6 3 3 4 9 8 0 10 6 4 1 2 1 8 1 4 0 6 9 3 3 2 3 0 0 5 8 8 3 0 1 8 0 5 3 7 0 5 3 0 6 7 2 1 8 0 5
55 313
1 0 2 0 0 0 2 2 1 2 0 1 0 1 0 0 1 1 1 2 0 0 1 2 1 0 2 0 1 1 0 0 0 1 0 0 0 0 0 1 2 0 1 0 0 0 2 2 0 2 0 1 272 2 2
55 253
6 6 9 1 8 6 0 0 6 9 0 1 1 4 6 6 4 5 8 10 5 0 9 0 9 9 1 0 2 10 0 3 1 2 2 2 10 9 9 3 10 7 6 5 5 7 7 4 1 6 3 0 5 3 2
57 338
0 2 1 280 0 1 1 2 1 1 2 1 0 1 1 2 2 0 1 1 0 2 0 1 0 1 0 0 2 2 1 0 1 1 2 0 2 1 2 2 2 2 2 0 0 2 1 2 1 2 0 1 1 0 1 1 0
10 15
2 1 2 2 1 2 3 0 0 2
83 374
1 2 0 0 1 2 2 0 1 2 2 2 0 1 2 2 293 0 1 2 1 2 0 0 0 0 0 0 2 0 0 0 2 2 2 0 2 2 0 0 1 2 1 1 0 0 1 0 1 1 2 2 0 1 2 0 1 2 1 0 0 2 0 0 1 2 1 1 0 1 2 2 2 1 1 0 0 1 2 2 1 0 2
10 13
2 0 3 2 0 0 1 1 3 1
10 16
0 10 0 1 0 1 2 0 0 2
72 385
6 4 1 3 4 4 9 5 8 9 2 6 8 5 4 3 5 8 10 10 2 2 0 9 10 10 3 2 9 10 1 7 6 6 5 6 4 9 10 0 6 10 5 5 9 4 0 9 7 5 5 8 6 7 4 0 0 7 1 5 7 2 3 7 2 4 2 3 9 0 8 10
56 455
1 1 2 2 2 2 2 0 2 409 2 0 0 2 0 2 1 2 1 0 0 1 1 1 0 2 0 2 2 2 1 1 2 0 1 1 0 2 1 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 1 0
