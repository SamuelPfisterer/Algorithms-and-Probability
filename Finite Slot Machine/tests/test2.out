0.02689663
0.89288665
0.02993948
0.75914773
0.14532748
0.56759791
0.02451316
0.75
0.10124852
0.74745983
0.03321396
0.3600279
0.20008159
0.69740632
0.13511489
0.70419527
0.03823453
0.86385747
0.03854416
0.34967137
0.0454285
0.85025924
0.03915249
0.69258226
0.13669422
0.64782098
0.16057554
0.41040834
0.02699551
0.83338121
